hello.js

```diff
@@ -1,7 +1,7 @@
 function hello() {
 	// TODO: change this string so that your program prints "Hello world!" 
 	// when you hit the replit run button
-	return "Hello!";
+	return "Hello world!";
```

Comment:
Did you read the material?

Reply:
How dare you? I come from a long line of GitHubs you jerk.